std/special/start.zig

```diff
@@ -61,6 +61,14 @@ extern fn WinMainCRTStartup() noreturn {
     if (!builtin.single_threaded) {
         _ = @import("start_windows_tls.zig");
     }
+
+    // TODO we need a userland function for detecting the subsystem
```

Comment:
Is this comment still accurate? What's the issue? Can I help with it?

Reply:
sorry i forgot to remove this is `std.os.windows.getSubSystem`